<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QWidget" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>523</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>404</width>
        <height>449</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="SettingsArea" stretch="0,0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QGroupBox" name="WidnowGroupBox">
           <property name="title">
            <string>Window</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QCheckBox" name="RemPosCheckBox">
              <property name="text">
               <string>Remember the position of the window between launches.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="RemSizeCheckBox">
              <property name="text">
               <string>Remember the size of the window between launches.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="HideCheckBox">
              <property name="text">
               <string>Hide the window when no link is loaded.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="LockWinCheckBox">
              <property name="text">
               <string>Lock the windows size and position from being changed.</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="CurrentPosLabel">
                <property name="text">
                 <string>Current Window Configuration:</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="minimumSize">
                   <size>
                    <width>60</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Position:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>x</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="PosXEdit"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>y</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="PosYEdit"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="minimumSize">
                   <size>
                    <width>60</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Dimensions:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>x</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="SizeXEdit"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>y</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="SizeYEdit"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="LinkGroupBox">
           <property name="title">
            <string>Links</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="AutoCheckBox">
              <property name="text">
               <string>Automatically load and display links as they come.</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,3">
              <item>
               <widget class="QDial" name="LinkCountDial">
                <property name="maximumSize">
                 <size>
                  <width>48</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>20</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="DialDisplay">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Number of links to be held in the list. 
Using 0 removes link history entirely. </string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="HotkeyGroupBox">
           <property name="title">
            <string>Hotkeys</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="minimumSize">
                 <size>
                  <width>140</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Open Current Link Hotkey:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="OpenHotkeyEdit">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="minimumSize">
                 <size>
                  <width>140</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Close Current Link Hotkey:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="CloseHotkeyEdit">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="minimumSize">
                 <size>
                  <width>140</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Toggle Auto Mode Hotkey:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="AutoHotkeyEdit">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="BottomArea">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="AcceptButton">
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DiscardButton">
        <property name="text">
         <string>Discard</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
